frontend/src/libs/components/button/button.tsx

```diff
@@ -1,11 +1,32 @@
+import { faChevronDown, faPlus } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
```

Comment:
we need to create custom icon wrapper for this library and reuse it here

Reply:
Did I understand correctly that all icon imports, for example `import { faChevronDown, faPlus } from '@fortawesome/free-solid-svg-icons';`, will be in the wrapper component? To use the wrapper do I need to create an enum for the icon names?

Reply:
I propose to add Icon component, which will have props like iconName, className for example
and this wrapper will be for svg icons. So you will have a map 'iconNameToSvg' - which will return svg icon by its name and actually this svg icon will return our Icon component.